contracts/ForkableBridge.sol

```diff
@@ -175,7 +175,7 @@ contract ForkableBridge is
     }
 
     /**
-     * @dev Allows the forkmanager to take out the forkonomic tokens
+     * @dev Allows to take out the forkonomic tokens
```

Comment:
Comment: Allows -> Allows anyone

Reply:
Come to think of it I would write this as
Allows anyone to take out their forkonomic tokens